docs/user/KubernetesVersions.md

```diff
@@ -31,44 +31,76 @@ export LOCATION="us-east1-b"
 gcloud container get-server-config --format "yaml(validMasterVersions)" --zone $LOCATION
 Fetching server config for us-east1-b
 validMasterVersions:
-- 1.18.16-gke.1200
-- 1.18.16-gke.500
-- 1.18.16-gke.302
-- 1.18.16-gke.300
-- 1.18.15-gke.1502
-- 1.18.15-gke.1501
-- 1.18.15-gke.1500
-- 1.18.15-gke.1102
-- 1.18.15-gke.1100
-- 1.18.14-gke.1600
-- 1.18.14-gke.1200
-- 1.18.12-gke.1210
-- 1.17.17-gke.3700
-- 1.17.17-gke.3000
-- 1.17.17-gke.2800
-- 1.17.17-gke.1500
-- 1.17.17-gke.1101
-- 1.17.17-gke.1100
-- 1.17.15-gke.800
-- 1.16.15-gke.12500
-- 1.16.15-gke.11800
-- 1.16.15-gke.10600
-- 1.16.15-gke.7801
-- 1.15.12-gke.6002
+- 1.25.6-gke.1000
+- 1.25.6-gke.200
+- 1.25.5-gke.2000
+- 1.24.9-gke.3200
+- 1.24.9-gke.2000
+- 1.24.9-gke.1500
+- 1.24.8-gke.2000
+- 1.23.16-gke.1100
+- 1.23.16-gke.200
+- 1.23.15-gke.1900
+- 1.23.15-gke.1400
+- 1.23.14-gke.1800
+- 1.23.14-gke.401
+- 1.23.13-gke.900
+- 1.22.17-gke.4000
+- 1.22.17-gke.3100
+- 1.22.16-gke.2000
+- 1.22.16-gke.1300
+- 1.22.15-gke.2500
+- 1.22.15-gke.1000
+- 1.21.14-gke.15800
+- 1.21.14-gke.14600
+- 1.21.14-gke.14100
+- 1.21.14-gke.8500
+- 1.21.14-gke.7100
 ```
 
 **NOTE**: This value will also be the value applied to the compute nodes of your custer.
 
-From here you would take one of these values and set the `kubernets_version` variable in your tfvars files like this:
+From here you would take one of these values and set the `kubernetes_version` variable in your tfvars files like this:
 
-```bash
-kubernetes_version = "1.18.15-gke.1102"
+```terraform
+kubernetes_version = "1.24.9-gke.3200"
```

Comment:
Why would this value of the `kubernetes_version` not be set to `1.24` as it was above?

Reply:
In this specific section we are showing the user that you can list out versions using `gcloud container get-server-config ...` and choose an exact one. We describe aliased versions (like 1.24), in the section right below this.

I could maybe change the sentence above on L63 to clear it up?

"To set the exact version for your cluster you would take one of these values and set the `kubernetes_version` variable in your tfvars files like this:"

Reply:
Updated the doc with the suggestion from my previous comment. 
@thpang ready for re-review,  let me know if clears up that section, I'm still willing to make changes.

Reply:
Yes, please update your comment to reflect the exact version.